tests/jsTests/menuSpec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Suite : Tests for showing and hiding of menu
+ * Description : This is a test suite for showing and hiding menu for smaller screens.
```

Comment:
Jasmine test suites have a syntax which covers comments as well. 'Describe' and 'it' are self-explanatory about the test suite. These comments were added because I received this as a review in the previous PR to add block comments. We have merged the previous PR using the same format. The standard way is to use 'describe' and 'it' for comments. Reference: [Jasmine Official Spec](https://github.com/jasmine/jasmine/blob/master/spec/core/CallTrackerSpec.js)